tutorials/473_py_tutorials/tutorial6_pandas.py

```diff
@@ -0,0 +1,120 @@
+# an example of a CSV. Feel free to open this file using a text editor in order to examine the 
+# structure of the file.
+
+# We could read in mm_data using the open and readline functions, but this would require us to 
+# manually separate the different values in each line and place them into separate arrays (this is
+# often referred to as "parsing" data). Using Pandas, we can read in the csv and place the data into 
+# a Pandas object called a "DataFrame" using a single command. 
+
+mm_df = pd.read_csv('mm_data', delim_whitespace=True)
+
+# Additional documentation on this function, including what to do if headers are not included in the
+# csv, is found here https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html
+# Note: Pandas can read several other file types other than CSV files, including Excel files.
+
+
+#---------------------------------------------------------------------------------------------------
+# Slicing DataFrames
+#---------------------------------------------------------------------------------------------------
+
+# To get an idea of what a DataFrame looks like, let's start by printing the DataFrame. Note how the
+# structure is similar to that of an Excel spreadsheet
+
+print(mm_df)
+print()
+
+# Now let's select certain columns (which are called "Series" in Pandas) in the DataFrame using 
+# the series names
+
+print(mm_df['T'])
+print()
+
+print(mm_df['RH'][:10])
+print()
+
+# We can also only select rows where a certain condition is true using the loc attribute. Here we 
+# only select the rows where qc is set to 0.
+
+print(mm_df.loc[mm_df['qc'] == 0])
+print()
+
+
+#---------------------------------------------------------------------------------------------------
+# Basic Computations in Pandas
+#---------------------------------------------------------------------------------------------------
+
+# Many computations such as mean, maximum, minimum, standard deviation, etc. can be performed in
+# Pandas. Note how for the relative humidity, a double percent sign is needed, otherwise Python
+# interprets % as a string formatting operator.
+
+print('Mean Temperature = %.2f deg C' % mm_df['T'].mean())
+print('Maximum Relative Humidity = %.2f %%' % mm_df['RH'].max())
+print('Standard Deviation of Wind Speed = %.2f m/s' % mm_df['spd'].std())
+print()
+
+
+#---------------------------------------------------------------------------------------------------
+# Adding New Series to a DataFrame
+#---------------------------------------------------------------------------------------------------
+
+# New series (columns) can be added to a Pandas DataFrame. For example, we might want another series
+# for potential temperature (theta) in Kelvin. We can define this new series using the existing
+# temperature and pressure series
+
+mm_df['theta'] = (mm_df['T'] + 273.15) * ((1000.0 / mm_df['p']) ** (287.04 / 1005.0))
+
+print(mm_df[:10])
```

Comment:
```suggestion
print(mm_df.head(10))
```
Perhaps not necessary here, but it might avoid weird interactions with the default indexer.